script.js

```diff
@@ -170,3 +170,31 @@ form.addEventListener('submit', (e) => {
   validateInputs();
 });
 // #endregion
+// Get All input fields and text area in the doc
+const inputFields = document.querySelectorAll('input, textarea');
+// Loop through each input field and add an "input" event listener
+inputFields.forEach((input) => {
+  input.addEventListener('input', (event) => {
+    // Get the updated value of the input field
+    const updatedValue = event.target.value;
+    // Get the key name for the local storage item (use the input field's name attribute)
+    const keyName = event.target.name;
+    // Save the updated value to local storage
+    localStorage.setItem(keyName, updatedValue);
+  });
+});
```

Comment:
- [ ] As stated in the project requirement Kindly I recommend you create a **single JavaScript object with all the data from the entire form** and save it in local storage. **Do not create one local storage entry per input field** You can refer  the below sample.
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```